src/main/resources/io/jenkins/plugins/coverage/targets/CoverageResult/index.jelly

```diff
@@ -62,7 +61,7 @@
                             coverageChartGenerator.generateSummaryChart(results, 'summaryChart', name)
                         });
 
-                        var element = '${it.childElement.displayName}';
+                        var element = '${it.childElement.name}';
```

Comment:
This does not like to be correct. Localized display names should be used in UI

Reply:
This part is adopted from Cobertura plugin before. But now, it is the name of `CoverageElement` and registered by others and we need to it as the key to save and restore coverage. So it cannot be Localized.

Reply:
It needs a follow-up JIRA task at least. There should be a way to localize the report layouts, but it can be done later

Reply:
OK, I have created one.